pkg/aspect/clean/windows.go

```diff
@@ -0,0 +1,35 @@
+//go:build windows
+
+package clean
+
+import (
+	"io/fs"
+	"syscall"
+	"time"
+)
+
+func (c *Clean) GetAccessTime(workspace fs.FileInfo) time.Duration {
+	winFileData := workspace.Sys().(*syscall.Win32FileAttributeData)
+
+	timeSinceAccess := time.Since(time.Unix(0, winFileData.LastAccessTime.Nanoseconds()))
+	timeSinceCreation := time.Since(time.Unix(0, winFileData.CreationTime.Nanoseconds()))
+	timeSinceModified := time.Since(time.Unix(0, winFileData.LastWriteTime.Nanoseconds()))
+
+	smallestTime := timeSinceAccess
+
+	if timeSinceCreation < timeSinceAccess && timeSinceCreation < timeSinceModified {
+		smallestTime = timeSinceCreation
+	} else if timeSinceModified < timeSinceAccess && timeSinceModified < timeSinceCreation {
+		smallestTime = timeSinceModified
+	}
+
+	return smallestTime
+}
+
+func (c *Clean) MoveFolderToTmp(dir string, name string) string {
+	return ""
+}
+
+func (c *Clean) ChangeFolderPermissions(folder string) ([]byte, error) {
+	return nil, nil
```

Comment:
is this still TODO?

Reply:
Yes it is still a TODO. Should I add a comment or make a ticket?

Reply:
yes, ideally both a ticket and a TODO here pointing to it. What's the behavior right now if you run on Windows? It should either not offer the new features at all (behave the same as `bazel clean`), or throw an error that it doesn't work. The worst case is for it to silently be broken.